spv/backend.go

```diff
@@ -95,9 +95,9 @@ func (s *Syncer) String() string {
 // interface.
 func (s *Syncer) LoadTxFilter(ctx context.Context, reload bool, addrs []dcrutil.Address, outpoints []wire.OutPoint) error {
 	s.filterMu.Lock()
-	if reload || s.rescanFilter == nil {
-		s.rescanFilter = wallet.NewRescanFilter(nil, nil)
-		s.filterData = nil
+	if reload || s.rescanFilter[s.filterWalletID] == nil {
+		s.rescanFilter[s.filterWalletID] = wallet.NewRescanFilter(nil, nil)
+		s.filterData[s.filterWalletID] = &blockcf.Entries{}
```

Comment:
The default value was previously `nil` rather than `blockcf.Entries{}`. Any reason for the change here?